product_assortment/__manifest__.py

```diff
@@ -1,11 +1,11 @@
-# Copyright 2018-2019 ACSONE SA/NV
```

Comment:
copyright should always contain at least the first year. The second is not relevant but 2018 should stay IMO.